client/package.json

```diff
@@ -27,13 +27,15 @@
     "vue-contenteditable": "^1.0.2",
     "vue-quill-editor": "^3.0.6",
     "vue-router": "^3.4.9",
+    "vue-the-mask": "^0.11.1",
```

Comment:
Since you added some packages, please bump up the app version by 0.0.1 also.

(It's a good habit to get into, as it helps us identify what version of the app is running.)

Reply:
To be clear, this project needs a bit more code to pass the version number to the footer for display, eg:

![image](https://user-images.githubusercontent.com/10002889/104758213-f10b2780-5712-11eb-9d5a-e8d8cc357e44.png)

I will probably get around to adding that code soon...